--- Makefile.orig	2005-02-03 14:41:19.000000000 +0100
+++ Makefile	2005-02-03 14:41:54.000000000 +0100
@@ -3,8 +3,8 @@
 
 # If you don't have it in /var/log/subsys, uncomment and define
 #CFLAGS+=-DLOCKFILE_DIR=\"/var/log\"
-LIBWRAP=-lwrap
+#LIBWRAP=-lwrap
 
 p910nd:	p910nd.c
-#	$(CC) -Wall $(CFLAGS) -o $@ p910nd.c
-	$(CC) -Wall $(CFLAGS) -DUSE_LIBWRAP -o $@ p910nd.c $(LIBWRAP)
+	$(CC) -Wall $(CFLAGS) -o $@ p910nd.c
+#	$(CC) -Wall $(CFLAGS) -DUSE_LIBWRAP -o $@ p910nd.c $(LIBWRAP)
